gatsby-node.js

```diff
@@ -76,7 +80,6 @@ const checkIsMdxOutdated = (path) => {
       intlMatch += match.replace(re, (_, p1, p2) => p1 + p2)
     })
   } catch {
-    console.error("regex error")
```

Comment:
I believe we still want this error log.

Reply:
Took this out after it was just producing a handful of non-specific errors in the build console. Will replace it with 
```console.error(`regex error in ${englishPath}`)```

Reply:
After chatting with @pettinarip decided to change this to a `console.warn` instead